docs/Migrating to Newer SDL Versions/Migrating SDL Core 6.1 to 7.0/index.md

```diff
@@ -0,0 +1,276 @@
+    </param>
+    <param name="capacity" type="Float" minvalue="0" maxvalue="1000000" mandatory="false">
+    <description>The absolute capacity of this fuel type.</description>
+    </param>
+    <param name="capacityUnit" type="Common.CapacityUnit" mandatory="false">
+    <description>The unit of the capacity of this fuel type such as liters for gasoline or kWh for batteries.</description>
+    </param>
+</struct>
+```
+
+Vehicle Data parameter `prndl` has been deprecated. Please make updates to use the new vehicle data struct `GearStatus`.
+
+```
+<struct name="GearStatus">
+    <param name="userSelectedGear" type="Common.PRNDL" mandatory="false">
+        <description>Gear position selected by the user i.e. Park, Drive, Reverse</description>
+    </param>
+    <param name="actualGear" type="Common.PRNDL" mandatory="false">
+        <description>Actual Gear in use by the transmission</description>
+    </param>
+    <param name="transmissionType" type="Common.TransmissionType" mandatory="false">
+        <description>Tells the transmission type</description>
+    </param>
+</struct>
+```
+
+## Additions
+
+### Vehicle Data
+
+- FuelRange was expanded to replace `fuelLevel` and `fuelLevel_state` parameters
+- New vehicle data type: GearStatus to replace `prndl` parameter
+- New vehicle data type: StabilityControlStatus
+- New vehicle data type: WindowStatus
+- New vehicle data type: HandsOffSteering
+
+It is not required to implement all vehicle data types. If a type is unsupported by your headunit, please be sure to respond to SDL Core with result `UNSUPPORTED_RESOURCE` if an unsupported request has been made.
+
+### HMI UI Additions
+
+#### Menu Changes
+
+SDL Core 7.0 adds extended capabilities to the app menu. SDL Core now supports nested submenus, dynamic menus, and menu browsing limitations while driver distraction mode is enabled. 
+
+Nested Submenus:
+
+An app can now request to add a submenu to another submenu by specifying a parentID. This did not require any new parameters on the HMI side. HMI's should be updated to process `parentID` in an AddSubMenu request. This param was previously only reserved for AddCommands. 
+
+```
+<function name="AddSubMenu" messagetype="request">
+...
+    <param name="menuParams" type="Common.MenuParams" mandatory="true">
+        <description>Position, parentID, and name of menu to be added.</description>
+    </param>
+...
+```
+
+Dynamic Menus:
+
+Two new RPCs were added to HMI API: UI.OnUpdateFile and UI.OnUpdateSubmenu. UI.OnUpdateFile request allows the HMI to request images from an SDL connected app when needed in an effort to reduce the amount of data an app needs to save on the headunit. UI.OnUpdateSubmenu request allows the HMI to dynamically request when submenu information is populated by the app. This functionality helps reduce the system load when an app first connects as the app is not required to load all menu contents onto the head unit immediately.
```

Comment:
Should code blocks be added around the RPCs here?

Also, "head unit" should be two words:
```suggestion
Two new RPCs were added to HMI API: UI.OnUpdateFile and UI.OnUpdateSubmenu. UI.OnUpdateFile request allows the HMI to request images from an SDL connected app when needed in an effort to reduce the amount of data an app needs to save on the head unit. UI.OnUpdateSubmenu request allows the HMI to dynamically request when submenu information is populated by the app. This functionality helps reduce the system load when an app first connects as the app is not required to load all menu contents onto the head unit immediately.
```